modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - New `expand` function added to `tinymce.selection` which expands the selection around the nearest word. #TINY-9001
 - New `expand` function added to `tinymce.dom.RangeUtils` to return a new range expanded around the nearest word. #TINY-9001
+- New `color_map_background` option which sets the default colors in the background highlight colorswatch. #TINY-9184
+- New `color_map_foreground` option which sets the default colors in the textcolor colorswatch. #TINY-9184
```

Comment:
In @hamza0867 now merged PR he combined these based on something @ltrouton said which makes sense to me, so this matches that (also removes some confusion around `default` related to the other PR).
```suggestion
- New `color_map_background` and `color_map_foreground` option which sets the base colors used in the `backcolor` and `forecolor` toolbar buttons and menu items. #TINY-9184
```